.github/workflows/oracle-workflow-test.yml

```diff
@@ -40,24 +40,25 @@ jobs:
   check:
     runs-on: ubuntu-latest
     outputs:
-      SKIP_CHECK: ${{ steps.check.outputs.SKIP_CHECK }}
+      SKIP_CHECK: ${{ steps.skip.outputs.SKIP_CHECK }}
     steps:
       - name: Checkout Action
         uses: actions/checkout@v3
 
       - name: Check if author has Quay.io credentials
+        id: skip
         env:
           QUAY_IO_USERNAME: ${{ secrets.QUAY_IO_USERNAME }}
           QUAY_IO_PASSWORD: ${{ secrets.QUAY_IO_PASSWORD }}
         run: |
-          if [[ $QUAY_IO_USERNAME != '' && $QUAY_IO_PASSWORD != '' ]]; then
-            echo "SKIP_CHECK=false" >> $GITHUB_OUTPUT
+          if [[ $QUAY_IO_USERNAME == '' && $QUAY_IO_PASSWORD == '' ]]; then
+            echo "SKIP_CHECK=true" >> $GITHUB_OUTPUT
           fi
 
   build:
     runs-on: ubuntu-latest
     needs: [check]
-    if: ${{ needs.check.outputs.SKIP_CHECK == 'false' || github.event_name == 'push' }}
+    if: ${{ needs.check.outputs.SKIP_CHECK != 'true' || github.event_name == 'push' }}
```

Comment:
Maybe better to use `github.event_name == 'pull_request'` so that the tests are run for PRs ... or event better to trigger it for both (can be written also as `contains(fromJSON('["push", "pull_request"]'), github.event_name)`)?

Reply:
@vjuranek So any changes to this workflow can be put to test only after it's in the `main` since it is using `pull_request_target`. Any other way to test it?

Reply:
@ani-sha just add you fork, debug/test it and once done remove it